lemur/common/celery.py

```diff
@@ -659,6 +673,10 @@ def certificate_rotate(**kwargs):
     log_data["message"] = "rotation completed"
     current_app.logger.debug(log_data)
     metrics.send(f"{function}.success", "counter", 1, metric_tags=metric_tags)
+
```

Comment:
Lets add a comment above this logic to make it clear why we are emitting this metric. 
```suggestion
# When doing rotation by source we want to emit a `last_success` metric for the given source. 
```